yarn.lock

```diff
@@ -4184,7 +4184,7 @@
   dependencies:
     "@types/react" "^17"
 
-"@types/react@*", "@types/react@>=16.9.11", "@types/react@^16", "@types/react@^17", "@types/react@^17.0.2":
+"@types/react@*", "@types/react@>=16.9.11", "@types/react@^17", "@types/react@^17.0.2":
```

Comment:
🤔 maybe a `yarn deduplicate` or something I need to do. Weird that the lock changed when I added nothing new